src/proc/y16i-to-y16y16.cpp

```diff
@@ -0,0 +1,42 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include "y16i-to-y16y16.h"
+#include "stream.h"
+// CUDA TODO
+//#ifdef RS2_USE_CUDA
+//#include "cuda/cuda-conversion.cuh"
+//#endif
+
+namespace librealsense
+{
+    struct y16i_pixel { uint16_t left : 16, right : 16;
+                        // left and right data are shifted 6 times so that the MSB 10 bit will hold the data
```

Comment:
Interesting..
We get the data as 10 bit LSB and output it as 10 bit MSB?
So for the user the first value after `0` (black) is 2^6 ?
Can you explain why do we do it? to have a [2^6 - 2^15] range?

Reply:
As we talked - OpenGL takes only the MSB byte - I have added it as comment in the code